common/build-style/zig-build.sh

```diff
@@ -0,0 +1,41 @@
+do_build() {
+	local zig_target
+	local zig_cpu
+	case $XBPS_TARGET_MACHINE in
+		aarch64-musl|i686-musl|x86_64-musl)
+			zig_target="${XBPS_TARGET_MACHINE%-musl}-linux-musl" zig_cpu="baseline";;
+		aarch64|i686|x86_64)
+			zig_target="${XBPS_TARGET_MACHINE}-linux-gnu" zig_cpu="baseline";;
+		armv6l-musl) zig_target="arm-linux-musl" zig_cpu="generic+v6";;
+		armv6l) zig_target="arm-linux-gnu" zig_cpu="generic+v6";;
+		armv7l-musl) zig_target="arm-linux-musl" zig_cpu="generic+v7a";;
+		armv7l) zig_target="arm-linux-gnu" zig_cpu="generic+v7a";;
```

Comment:
I wonder if for all archs you couldn't do something like `armv7l*)` then have a conditional for `XBPS_TARGET_LIBC` to append `-linux-gnu` or `-linux-musl`?

Would be a tad cleaner.